test/component_test.go

```
@@ -0,0 +1,168 @@
+	assert.Equal(s.T(), fmt.Sprintf("Access role for %s", bucketID), *awsRole.Description)
+
+	assert.EqualValues(s.T(), 3600, *awsRole.MaxSessionDuration)
+	assert.Equal(s.T(), "/", *awsRole.Path)
+
+	s.DriftTest(component, stack, nil)
+}
+
+func (s *ComponentSuite) TestEnabledFlag() {
+	const component = "tfstate-bucket/disabled"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
```

golangci-lint (1.62.2)

const `awsRegion` is unused

(unused)